backend/controllers/passController.js

```diff
@@ -20,7 +20,7 @@ const generatePass = asyncHandler(async (req, res) => {
         email,
         duration,
         start,
-        checkInStatus: true,
+        checkInStatus: false,
         generateId: user.id,
```

Comment:
Which is better?
`generateId` or `generateById` or `generatedUserId`

Reply:
generatedUserId